core/docker-entrypoint.sh

```diff
@@ -37,9 +87,12 @@ fi
 
 if [ "$(id -u)" = '0' ] && [ "${QUESTDB_DATA_DIR%/}" != "/root/.questdb" ] && [ "$RUN_AS_ROOT" = "false" ] ; then
     echo "Running as questdb user"
-    find_and_own_dir ${QUESTDB_DATA_DIR}
+    find_and_own_dir $(id -u questdb) $(id -g questdb)
```

Comment:
could you have a look at clickhouse entry point, it flows better: https://github.com/ClickHouse/ClickHouse/blob/master/docker/server/entrypoint.sh

Reply:
it would be good to define UID and GID we use for the whole thing at the top of the script and then do things systematically from there. For example clickhouse have explicit `DO_CHOWN` flag, which i think is a good idea.